tests/integration_test.go

```diff
@@ -25,49 +32,123 @@ func TestHappyPath(t *testing.T) {
 		"MINIO_ACCESS_KEY=" + accessKeyID,
 		"MINIO_SECRET_KEY=" + secretAccessKey,
 	}
+
+	var stdBuffer bytes.Buffer
+	mw := io.MultiWriter(os.Stdout, &stdBuffer)
+
 	err = minios.Start()
 	if err != nil {
 		t.Fatal("minio error:", err)
 	}
-	t.Log("minio server:", minios.String())
+	t.Log("server:", minios.String())
 
 	defer minios.Process.Kill()
 
-	minioc := exec.Command("mc", "config", "host", "add", "mycloud", "http://127.0.0.1:9000", accessKeyID, secretAccessKey)
+	// minioc := exec.Command("mc", "config", "host", "add", "narcloud", "http://127.0.0.1:9000", accessKeyID, secretAccessKey, "--config-dir", ".", "--api", "s3v4")
+
+	minioc := exec.Command("mc", "alias", "set", "narcloud", "http://127.0.0.1:9000", accessKeyID, secretAccessKey, "--api", "s3v4")
```

Comment:
this command seems to be not supported by the version of minio that is pulled into the environment